insights/specs/default.py

```diff
@@ -494,6 +494,15 @@ def semid(broker):
     keystone_crontab = simple_command("/usr/bin/crontab -l -u keystone")
     keystone_crontab_container = simple_command("docker exec keystone_cron /usr/bin/crontab -l -u keystone")
     keystone_log = first_file(["/var/log/containers/keystone/keystone.log", "/var/log/keystone/keystone.log"])
+
+    uname_r = simple_command("/usr/bin/uname -r")
+
+    @datasource(uname_r, context=HostContext)
```

Comment:
The `uname_r` is not collected and it's already covered by the `uname` spec (`uname -a`), please change to use the existing `Uname` parser which is based on `uname` spec here. e.g.

```
from insights.parsers.uname import Uname
@datasource(Uname, context=HostContext)
  un = broker[Uname]
  return "/var/lib/kpatch/" + un.kernel
```

Reply:
Thanks for the tip, I do it this way in the latest patch